### ISSUE TITLE
D3

### ISSUE THREAD

**Issue by aharris88** [aharris88/awesome-snippets#5]
I would like an awesome list of D3 snippets.

**Comment by AmGarera** [comment]
Label this for hacktoberfest and I'll get right on it 👍

**Comment by aharris88** [comment]
@AmGarera There you go.

**Comment by AmGarera** [comment]
Submitted a PR for this. Ref #8

**Comment by puripant** [comment]
Just wonder if this is what you are looking for? https://github.com/wbkd/awesome-d3#code-editors

**Comment by agarrharr** [comment]
Yep, those 3 look pretty good (except that the sublime-text-2 one might be out of date).
